wiki/Article_Style_Guide/en.md

```diff
@@ -370,104 +416,127 @@ Please limit to a level of four (4) sub-lists at a time (if you need more, you s
 
 ## Code
 
-Use code (inline) when noting:
-
-- a key on the keyboard
-  - should use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
-  - examples:
-    - `.`
-    - `Alt`
-    - `Ctrl` + `Shift` + `A`
-- the text of a button
-  - examples:
-    - `1. Let's do it.`
-    - `4. Noooo! I didn't mean to!`
-- the name of a folder
-  - examples:
-    - `Exports`
-    - `Downloads`
-- the name of a directory
-  - examples:
-    - `/Songs/`
-    - `/Skins/User/`
-- the name of a keyword
-  - examples:
-    - `ComboBurstStyle`
-    - `[Events]`
-- the name of a file extension
-  - must add a period (`.`) then the file extension in lowercase letters.
-  - examples:
-    - `.jpg`
-    - `.gif`
-    - `.osz`
-- the name of a chat channel
-  - examples:
-    - `#lobby`
-    - `#osu`
+Any key on the keyboard should use inline code.
+When doing so, it should use capital letters for single characters and [camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+For example:
+
+- `` `.` ``
+- `` `Alt` ``
+- `` `Ctrl` + `Shift` + `A` ``
+
+When referring to the text of a button or menu, use inline code.
+For example:
+
+- `` Click `File` then `Save As...` ``
+- `` `1. Let's do it.` ``
+- `` `4. Noooo! I didn't mean to!` ``
+
+When referring about the name of a folder, use inline code.
+For example:
+
+- `` `Exports` ``
+- `` `Downloads` ``
+
+When referring to the name of a directory, use inline code.
+For example:
+
+- `` `Downloads/` ``
+- `` `Skins/User/` ``
+
+When referring to the name of a keyword or command used in _osu!_ or of any application, use inline code.
+For example:
+
+- `ComboBurstStyle`
+- `[Events]`
+- `git --help`
+
+Any file extension should use inline code.
+A period (`.`) should be prefixed followed by the file extension in lowercase letters.
+
+- `.jpg`
+- `.gif`
+- `.osz`
+
+The name of a chat channel should use inline code.
+For example:
+
+- `#lobby`
+- `#osu`
 
 ### Code Blocks
 
 When using code blocks, use the ` ``` ` (triple grave mark) syntax.
 
+Code blocks in the osu!wiki does **not** do syntax highlighting.
+
 The osu!wiki site has a feature where you _could_ use four spaces to trigger the code blocks; however, this is **heavily** discouraged.
```

Comment:
The most emphasis in one sentence in all of the article. Although it is sensible, it is a bit out of place. Consider using italics to emphasise in sentences a bit more frequently.